extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -121,7 +125,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 
 	GoogleTag.prototype.addSlot = function (adElement) {
 		var sizes = adElement.getSizes(),
-			slot = slots[adElement.getId()];
+			slot = slots[adElement.getSlotPath()];
```

Comment:
I changed it to slotPath because we can access slotPath from google's slot, I couldn't find an API to get the adUnit (our adunit, not google's).